### ISSUE TITLE
mean_iu is not good enough

### ISSUE THREAD

**Issue by worstprogrammerCN** [shekkizh/FCN.tensorflow#98]
I trained the model and calculated the miou(mean intersection over union)myself, but the miou is just around 19. According to the baseline in this page http://sceneparsing.csail.mit.edu, the baseline of miou of Fcn8s is 24.
I have to do sth. to improve the miou. I have set the learning rate to 1e-6.
 I want to know if the author has tested the miou？What can I do to improve my miou，still using FCN-8s？
I'll be grateful to anyone who helps me！

**Comment by greenhornno1** [comment]
hi , I want to test my model mean_iu as well 
I am totally new for tensorflow so i dont know how to edit the code ,
can you share yours ? Thank you
